src/Modules/Compensator.cpp

```diff
@@ -69,7 +69,7 @@ extern struct Params Config;
                 upsampled_states (size := imus.size): before t1 and after t2
         */
         States Compensator::upsample(const States& states, const IMUs& imus) {
-            assert (imus.front().time <= states.front().time and states.back().time <= imus.back().time);
+            //assert (imus.front().time <= states.front().time and states.back().time <= imus.back().time);
```

Comment:
I had to comment this out. The first part of the assert would always throw an error. I think it has to change from `<=` to `>=` to be as intended. Not sure how it works in the ROS1 version.

Issue created about it #28